pymc_marketing/clv/utils.py

```diff
@@ -175,6 +176,10 @@ def _find_first_transactions(
         Time granularity for study.
         Default: 'D' for days. Possible values listed here:
         https://numpy.org/devdocs/reference/arrays.datetime.html#datetime-units
+    sort_transactions: bool, optional
+        Default: True
+        If encountering memory issues, set to False and order raw customer data in upstream external system
```

Comment:
```suggestion
        May increase memory usage
```

Reply:
I think a detailed docstring is needed here because this sorting operation is necessary, but can be skipped if already pre-sorted upstream in the data pipeline.

Reply:
I found the original sentence hard to understand. Maybe say something like "May increase memory usage. Not needed if data is already sorted"?

Reply:
Edited into 
```
If raw date is already sorted in chronological order, set to `False` to improve computational efficiency.
```